NEWS.md

```diff
@@ -1,3 +1,7 @@
+# hubValidations 0.0.0.9006
+
+* `validate_pr()` now checks for deletions of previously submitted model metadata files and modifications or deletions of previously submitted model output files, adding an `<error/check_error>` class object to the function output for each detected modified/deleted file (#43 & #44).
```

Comment:
After some thinking, I wonder if it makes sense to have the possibility (maybe in a future version) to set this check to returns a warning instead of an error. I am particularly thinking about the Scenario projection rounds, where during an ongoing round, a team might want to submit an update of their projections.

Reply:
So a warning would still get flagged and fail the workflow. The error just made it a bit more visually obvious when printing the results. 

Do you want it to be able to pass validation?

Reply:
That's a good point, I guess if it's in the ongoing round, it will make sense for us to have it pass the validation, can it be a parameter the hub set? so that depending on the hub they can decide if they want this type of issue to fail or pass the validation (with just a warning/message)

Reply:
Yes will do that! thanks